logitech_r400/README.md

```diff
@@ -18,28 +18,39 @@ You'll need pyusb 1.0+ for this driver to work. To install it:
 sudo pip install --upgrade --pre pyusb
 ```
 
-### Accessing without sudo
-I found those informations in the following [wiki](http://www.tincantools.com/wiki/Accessing_Devices_without_Sudo) to access the Logitech R400 without sudo.
-
-- first make sure you're in the plugdev group.
-- then create a udev file:
+### Config setup
+First, find out pointer's vendor and product values. To do that, in the console run `dmesg -w` and plug in the pointer's dongle to the usb port. A message similar to the one below should appear:
 
 ```
-sudo nano /etc/udev/rules.d/89-logitech-r400.rules
+[64802.439385] usb 1-5.3: new low-speed USB device number 21 using xhci_hcd
+[64802.604109] usb 1-5.3: New USB device found, idVendor=2571, idProduct=4101
+[64802.604116] usb 1-5.3: New USB device strings: Mfr=1, Product=2, SerialNumber=0
+[64802.604120] usb 1-5.3: Product: HAS HS304
+[64802.604124] usb 1-5.3: Manufacturer: HAS  
+[64802.612420] input: HAS   HAS HS304 as /devices/pci0000:00/0000:00:14.0/usb1/1-5/1-5.3/1-5.3:1.0/0003:2571:4101.003A/input/input90
+[64802.671856] hid-generic 0003:2571:4101.003A: input,hidraw1: USB HID v1.11 Keyboard [HAS   HAS HS304] on usb-0000:00:14.0-5.3/input0
+[64802.675745] input: HAS   HAS HS304 as /devices/pci0000:00/0000:00:14.0/usb1/1-5/1-5.3/1-5.3:1.1/0003:2571:4101.003B/input/input91
+[64802.735835] hid-generic 0003:2571:4101.003B: input,hidraw2: USB HID v1.11 Mouse [HAS   HAS HS304] on usb-0000:00:14.0-5.3/input1
 ```
 
-- Add the following udev rule to that file:
+In the example above, you can see there are `idVendor` and `idProduct` values present, these are the ones you're interested in.
+
+Now, make sure you're in the `plugdev` group and create a udev file:
 
 ```
-ATTRS{idProduct}=="0111", ATTRS{idVendor}=="0e00", MODE="666", GROUP="plugdev"
+sudo nano /etc/udev/rules.d/89-logitech-r400.rules
 ```
-or
+
+Add the following udev rule to that file:
+
 ```
-ATTRS{idProduct}=="c538", ATTRS{idVendor}=="046d", MODE="666", GROUP="plugdev"
+ATTRS{idProduct}=="<product_value>", ATTRS{idVendor}=="<vendor_value>", MODE="666", GROUP="plugdev"
 ```
-(try both - which is succesfull depends on exactly what device you have)
+where `<product_value>` and `<vendor_value>` are values found before.
 
-- Reload the rules without rebooting:
+Now, reload the rules without rebooting:
 ```
 sudo udevadm trigger
 ```
+
+Information based on the following [wiki](http://www.tincantools.com/wiki/Accessing_Devices_without_Sudo).
```

Comment:
You should also add a section to tell the user to check the config.yaml and make sure the Product and Vendor of their pointer are declared in the yaml.